viaf.py

```diff
@@ -136,7 +136,7 @@ def weigh_contributor(cls, candidate, working_sort_name, known_titles=None, stri
         # the given author, make sure that one of the working names
         # shows up in a name record.
         if strict:
-            if len(match_confidences) == 0:
+            if not match_confidences:
```

Comment:
Now that I know what's going on, I think your change is fine. Without knowing what's going on this sort of change always sets off alarm bells.